modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -34,6 +34,7 @@ const VECTOR_TO_POINT_MATRIX = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0];
 const IDENTITY_MATRIX = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1];
 const DEFAULT_PIXELS_PER_UNIT2 = [0, 0, 0];
 const DEFAULT_COORDINATE_ORIGIN = [0, 0, 0];
+const EXPERIMENTAL_LNG_LAT_ZOOM_THRESHOLD = 12;
```

Comment:
Thoughts about the magic threshold:
- We need some comments on how this number is determined. At zoom level 12, viewport height covers `180 / Math.pow(2, 12) ~= 0.0439 (degrees)`. According to the tests [here](https://github.com/uber-common/viewport-mercator-project/blob/master/docs/articles/offset-accuracy.md) the lngLat offset mode can handle at least another zoom level with errors less than 0.1 pixels.
- We can easily add some test cases to compare projection results right below and above this threshold. The goal is that during smooth zoom in/out there is no visible "jump" for the same coordinate.

Reply:
There is currently no visible jump.
I think we can probably go a little bit more like maybe 14.

Reply:
My point being, this logic should be captured by some form of documentation or test case. Both accuracy and discontinuity can be measured quantitively. Arbitrary manual test case does not support the selection of a "magic number". E.g. lnglat offset mode's error increases with latitude. Our test datasets are very SF centric.

Reply:
OK, let me try to do that and come up with some numbers.

Reply:
Any help on how `offset-accuracy.md` was calculated?